src/routes/tools/extreme-precipitation/_ChartTitle.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
```

Comment:
Wondering if we should make these Title components a Partial that have slots for the h2 and p elements. That way we wouldn't need to be rewriting the Change Location Button each time we need it.

Reply:
The partial might work for some tools. But others e.g. Wildfire, Snowpack have more complex Title components with multiple `p` elements and notifications.

Reply:
True, though you could still wrap multiple `<p>` elements in a `div` to slot them. It brings up a good point though which is that standardizing these components is difficult!